omnibus-base-mainnet-andromeda.toml

```diff
@@ -1,38 +1,33 @@
 name = "synthetix-omnibus"
 version = "3.3.5"
-description = "Andromeda dev deployment"
+description = "Andromeda deployment"
 preset = "andromeda"
 include = [
     "tomls/core.toml", # Provision the core system
     "tomls/settings.toml", # Apply minimum liquidity ratio and withdrawal timeout
     "tomls/permissions.toml", # Allow deployer to create pools
-    "tomls/omnibus-base-mainnet-andromeda/permissions.toml", # Set deniers, etc.
     "tomls/pools/spartan-council.toml", # Create Spartan Council Pool
-    # Collaterals
-    "tomls/omnibus-base-mainnet-andromeda/collaterals/susdc.toml",
-    "tomls/collaterals/susdc.toml",
-    # Factories
     "tomls/markets/spot-factory.toml",
     "tomls/markets/perps-factory.toml",
-    # Spot Markets
     "tomls/omnibus-base-mainnet-andromeda/spot/usdc.toml", # sUSDC spot market/wrapper
-    # Perps Markets
+    "tomls/omnibus-base-mainnet-andromeda/permissions.toml", # Set deniers, etc.
+    "tomls/permit-all-perps-perpsSystem.toml", # Enable perps market
     "tomls/omnibus-base-mainnet-andromeda/perps/referrers.toml",
     "tomls/omnibus-base-mainnet-andromeda/perps/global.toml",
+    "tomls/oracles/perps-keeper-cost.toml", # Add gas oracle for keeper fees
+#    "tomls/omnibus-base-mainnet-andromeda/perps/feeCollector.toml", ?? commented about until SIP-345 released
+    "tomls/omnibus-base-mainnet-andromeda/collaterals/susdc.toml",
+    "tomls/collaterals/susdc.toml",
     # BTC
     "tomls/markets/perps/btc.toml",
+    "tomls/oracles/pyth-btc.toml",
     "tomls/omnibus-base-mainnet-andromeda/perps/btc.toml",
     # ETH
     "tomls/markets/perps/eth.toml",
-    "tomls/omnibus-base-mainnet-andromeda/perps/eth.toml",
-    # Oracles
-    "tomls/oracles/pyth-btc.toml",
     "tomls/oracles/pyth-eth.toml",
-    "tomls/oracles/perps-keeper-cost.toml", # Add gas oracle for keeper fees
-    # "tomls/oracles/pyth-snx.toml",
-    # Permissions
-    "tomls/permit-all-perps-perpsSystem.toml", # Enable perps market
-
+    "tomls/omnibus-base-mainnet-andromeda/perps/eth.toml",
+#    # SNX ?? does not exist on mainnet
```

Comment:
@kaleb-keny i might be misundertanding by the pyth oracle contract on base seems to be https://basescan.org/address/0x8250f4aF4B972684F7b336503E2D6dFeDeB1487a and the toml seems to exist

Reply:
This was not included in the mainnet release, but was included in the goerli release